src/ui/bibliography-renderer/render-list-content-script.ts

```diff
@@ -0,0 +1,74 @@
+import { Reference } from "../../model/reference.model";
+
+declare const webviewApi: any;
+
+export default function(_context) { 
```

Comment:
It should be `context`. The `_` is only there when the variable is not actually used (it's to silence the TypeScript error).

Reply:
Got it